cmd/captplanet/setup.go

```diff
@@ -30,7 +30,7 @@ type Config struct {
 	APIKey              string `default:"abc123" help:"the api key to use for the satellite"`
 	EncKey              string `default:"highlydistributedridiculouslyresilient" help:"your root encryption key"`
 	Overwrite           bool   `help:"whether to overwrite pre-existing configuration files" default:"false"`
-	MinioTLS            bool   `default:"false" help:"generate sample TLS certs for Minio GW"`
+	GenerateMinioCerts  bool   `default:"false" help:"generate sample TLS certs for Minio GW"`
```

Comment:
Ah, wait this is also for the Config, not just the setup? Sorry, can you revert this. I just realized the previous one was probably better.

Reply:
What do you mean for Config? It will be generating certs during setup only so you version fits quite well.

Reply:
Oh, well. For some reason it seemed wrong. Whichever you think is better.